docs/third_party.rst

```diff
@@ -139,6 +139,9 @@ period ask to raise the status.
 - `aiohttp-validate <https://github.com/dchaplinsky/aiohttp_validate>`_
   Simple library that helps you validate your API endpoints requests/responses with json schema.
 
+- `aiohttp-pydantic <https://github.com/Maillol/aiohttp-pydantic>`_
+  An aiohttp.View to validate request body, qs and headers regarding function annotations and generate Open API doc. Python 3.8+ required.
```

Comment:
Looks like the spellchecker doesn't like `qs`. Could you please change it? We need CI to be green before merging.

Reply:
Sorry, I fixed that typo and added a change file in order to Timeline protection changes to green.

Reply:
Is SAP SE the owner of aiohttp? The CLA assistant ask me to sign your License Agreement with SAP Se.

Reply:
My fault. The CLA text was fixed.
Now it is pretty close to the Python CLA.
The reference is here: https://gist.github.com/asvetlov/dbce6679052f5c89d3d13f501107c1d4

Sorry for misleading.

Reply:
Ok great! Now the received link point to right text. Thanks